### PR TITLE
Trio process fixes

### PR DESCRIPTION
**Pull request by apregier**
These changes are needed to get consistent process outputs from the cle regression test process.

### CODE REVIEW

File: lib/perl/Genome/Interfaces/Comparable/Command/Diff.pm [hunk 1]
```diff
@@ -58,6 +58,7 @@ sub diffs_message {
     my $self = shift;
     my $diffs = $self->_diffs;
     my $diff_string = "DIFFERENCES FOUND:\n";
+    $diff_string .= sprintf(" Comparing new object %s to blessed object %s\n", $self->new_object->id, $self->blessed_object->id);
```
Comment (tmooney) [inline lib/perl/Genome/Interfaces/Comparable/Command/Diff.pm]: It's a little strange that executing this command will print this line twice if there are diffs.

Comment (apregier) [inline lib/perl/Genome/Interfaces/Comparable/Command/Diff.pm]: Yeah, the main thing is that this one will make it into the e-mail, whereas the other one is just a status message and will only be in the jenkins console log

Comment (tmooney) [inline lib/perl/Genome/Interfaces/Comparable/Command/Diff.pm]: Fair enough!  It is nice to know the info in both places :smile:

**Comment by tmooney** [comment]
+1.

It looks like the `AlignmentStatsSummary` and `CoverageStatsSummary` commands might be ripe for deduplication!

**Pull request merged**